app/uk/gov/hmrc/transitmovementsvalidator/models/MessageType.scala

```diff
@@ -51,11 +51,18 @@ object MessageType {
   /** E_PRE_NOT (IE170) */
   case object PresentationNotificationForPreLodgedDec extends DepartureMessageType("IE170", "CC170C", "/xsd/cc170c.xsd", "/json/cc170c-schema.json")
 
+  case object ReleaseForTransit extends DepartureMessageType("IE029", "CC029C", "/xsd/cc029c.xsd", "/json/cc029c-schema.json")
+
+  case object GoodsReleaseNotification extends ArrivalMessageType("IE025", "CC025C", "/xsd/cc025c.xsd", "/json/cc025c-schema.json")
+
+  case object UnloadingPermission extends ArrivalMessageType("IE043", "CC043C", "/xsd/cc043c.xsd", "/json/cc043c-schema.json")
+
```

Comment:
myself and @derekspalmer want to have a tests for this messages - yes, it's better we support for other messages as well and maybe we should create a TechDebt ticket for this

Reply:
I'm guessing you wanted to test these because of the "minOccurs" thing?

As long as these message types aren't accepted for submission to the entire API, then I'm okay with this here. If adding them here allows them to be submitted, we should fix that in the API itself. I don't think we do, but please check that we can't.

Reply:
yep, that's right - it allows to attach a message with this messageType.
I think the simplest thing to do is to remove these here in MessageType as you have suggested.

Reply:
Yep, if we want these back (and I'm not against that really), we can always create a tech debt ticket. I can see the use of having such message types in here, if only to support traders that want to reuse our validator.